lib/cli/src/versions.ts

```diff
@@ -0,0 +1,58 @@
+// auto generated file, do not edit
```

Comment:
@shilman I did not see ANY references of the script that generates this (but the file is up to date, so I assume something is using it?)

Could you let me know if another script is generating this?